guides/react_checkout_guide/09_back_to_the_frontend.md

```diff
@@ -0,0 +1,118 @@
+## Go back to the frontend
+
+Now we want to go back to our frontend and add the changes there as well. We need to make sure that the order totals are showing up.
+
+As always we first start with our tests.
+
+```js
+cy.get("button")
+  .contains("View order")
+  .click();
+cy.get("#order-details").within(() => {
+  cy.get("li")
+    .should("have.length", 2)
+    .first()
+    .should("have.text", "1 x Salad")
+    .next()
+    .should("have.text", "1 x Ice Cream");
+});
```

Comment:
Refactor your test by adding  the following section to your it block in your test file. 

```js
[....]
		})

		cy.get('button').contains('View order').click()

		cy.get('#order-details').within(() => {
			cy.get('li')
				.should('have.length', 2)
				.first().should('have.text', '1 x Salad')
				.next().should('have.text', '1 x Ice Cream')
		})

		cy.get('button').contains('View order').click()

		cy.get('#order-details').should('not.exist')

	});
```